pkg/controller/gitopsconfig/controller_test.go

```diff
@@ -627,3 +645,21 @@ func findJobList(cl client.Client) (int, error) {
 	}
 	return len(jobList.Items), nil
 }
+
+func findRunningJob(cl client.Client) batch.Job {
+	// At times other jobs can exist
+	jobList := &batchv1.JobList{}
+	// Looking up all jobs
+	err := cl.List(context.TODO(), &client.ListOptions{
+		Namespace: namespace,
+	}, jobList)
+	if err != nil {
+		log.Error(err, "unable to list jobs")
+		return batch.Job{}
+	}
+	// Return the first instance that is a delete job
```

Comment:
This comment says about a delete job, are you sure this is correct?

Reply:
fixed the comment